src/invite/InviteModal.tsx

```diff
@@ -46,7 +48,7 @@ const InviteModal = ({
   }
 
   return (
-    <SafeAreaView style={styles.container}>
+    <SafeAreaView style={[styles.container, { height, width }]}>
```

Comment:
Curious, do we even need the explicit width and height? Wouldn't it try to fill the available space?

Reply:
it is position absolute and doesn't seem to play nice without the width and height....
<img src="https://user-images.githubusercontent.com/20150449/227983969-217c79ab-3b8c-4ac3-ae10-184eca696ef7.png" width=200 />